pkg/client/restclient/client.go

```diff
@@ -87,7 +87,7 @@ func NewRESTClient(baseURL *url.URL, versionedAPIPath string, config ContentConf
 		config.GroupVersion = &unversioned.GroupVersion{}
 	}
 	if len(config.ContentType) == 0 {
-		config.ContentType = "application/json"
+		config.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
oh.. I though about "get" and this should work, because of renegotiation. But you're right, that it will fail for post and put.

Will fix that. (but we are waiting for other PRs anyway first :))